README.md

```diff
@@ -33,9 +33,15 @@ Set environment variables.
 export KUBECONFIG=</path/to/hub_cluster/kubeconfig>
 ```
 
-Override the docker image (optional)
+Build the docker image.
```

Comment:
build the docker image to run the sample addon.

Reply:
Done